packages/tangle/tangle.go

```diff
@@ -292,6 +294,7 @@ type Options struct {
 	WeightProvider               WeightProvider
 	SyncTimeWindow               time.Duration
 	StartSynced                  bool
+	ForceCacheTime               database.CacheTimeProvider
```

Comment:
Name the field as the type `CacheTimeProvider` because `ForceCacheTime` is an implementation detail hidden inside the `database.CacheTimeProvider` type and in the future, we might apply different logic for cache management, but Tangle should know nothing about that.

Also, use a pointer as in any other place. I understand your intention to use value to gracefully handle the case there is no `CacheTimeProvider` passed, but it's very implicit and hard to reason about. Instead, we should use a _default_ `CacheTimeProvider` instance when no one passed such option as it's done for other options in the Tangle.